src/components/Forms/Register.jsx

```diff
@@ -35,6 +36,22 @@ const Register = () => {
   };
 
   const handleSubmit = () => {
+    const incompleteFields = Object.values(user).some(
+      (value) => value === "" || !value
+    );
+    const invalidRequirements = Object.values(requirements).some(
+      (check) => !check.state
+    );
+    const atLeastOneDiet = Object.values(dietaryRestrictions).some(
```

Comment:
this field is not required, since u could have no diet restrictions

Reply:
but we have a checkbox field called None for diet restrictions